src/components/expense-list-section/expense-list/expense-list-header/expense-list-header.css

```diff
@@ -0,0 +1,100 @@
+.expense-list-header {
+    display: flex;
+    gap: 20px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+@media (max-width: 991px) {
+    .expense-list-header {
+        max-width: 100%;
+        flex-wrap: wrap;
+    }
+}
+
+.expense-list-header {
+
+    .list-name {
+        color: #191919;
```

Comment:
css variable

Reply:
in other branch I will fix it 
https://github.com/ofir-zeitoun/expenses-client/issues/33